cluster-provision/k8s/1.30/manifests/local-volume.yaml

```diff
@@ -113,7 +113,7 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
           - name: JOB_CONTAINER_IMAGE
-            value: "quay.io/external_storage/local-volume-provisioner:v2.3.2"
+            value: "quay.io/external_storage/local-volume-provisioner:v2.5.0"
```

Comment:
Can we apply this bump to all three providers in a separate PR?

Reply:
@brianmcarey a new PR created #1285